src/assets/js/product.js

```diff
@@ -20,6 +22,21 @@ class Product extends BasePage {
         }
     }
 
+    initProductOptionValidations() {
+      const productForm = document.querySelector('.product-form');
+
+      productForm?.addEventListener('change', async () => {
+        const productOptions = productForm.querySelector('salla-product-options');
+
+        if (!productOptions) {
+          salla.product.getPrice(new FormData(productForm))
+        } else {
+          const isValid = await productOptions?.reportValidity();
```

Comment:
why you are calling the reportValidity on the productOptions?
call it on the form and remove the await and the async